### PR TITLE
[wip] Debian and RPM packaging

### PR DESCRIPTION
**Pull request by neiljerram**
@dims Here is the packaging I said I might work on.  I'm not sure it's right yet - even for my own purposes - because I don't yet have successful package builds - but you may already like to take a look and let me know if you have any comments.

### CODE REVIEW

File: rpm/etcd3-gateway.spec [hunk 1]
```diff
@@ -0,0 +1,45 @@
+%{!?python_sitelib: %define python_sitelib %(%{__python} -c "from distutils.sysconfig import get_python_lib; print get_python_lib()")}
+
+Name:           etcd3-gateway
+Summary:        etcd3 gateway Python Client
+Epoch:          1
```
Comment (apevec) [inline rpm/etcd3-gateway.spec]: Why is that?
There is no need to start with Epoch 1 for a new package

File: rpm/etcd3-gateway.spec [hunk 2]
```diff
@@ -0,0 +1,45 @@
+%{!?python_sitelib: %define python_sitelib %(%{__python} -c "from distutils.sysconfig import get_python_lib; print get_python_lib()")}
+
+Name:           etcd3-gateway
```
Comment (apevec) [inline rpm/etcd3-gateway.spec]: python-$MODULE is recommended naming for RPM Python modules
https://docs.fedoraproject.org/en-US/packaging-guidelines/Naming/#_python_modules

File: rpm/etcd3-gateway.spec (hunk 1 above)
Comment (neiljerram) [reply rpm/etcd3-gateway.spec]: Sorry, ignorance on my part.

**Comment by neiljerram** [comment]
Thanks @apevec for your comments.  I will close this PR now anyway, as I had forgotten about it, and it sounds like there is a more official process in the offing.

**Pull request closed without merging**